src/containers/NotificationsContainer.js

```diff
@@ -1,21 +1,21 @@
 import React from "react";
 import { connect } from "react-redux";
 import { Redirect } from "react-router-dom";
-import { fetchUser } from "../actions/useractions";
+import { fetchUserNotifications } from "../actions/useractions";
 import Notification from "../components/Notifications/Notification.js";
 // import {Feed, Container} from 'semantic-ui-react'
 class NotificationsContainer extends React.Component {
   componentDidMount = () => {
-    this.props.fetchUser();
+    this.props.fetchUserNotifications();
   };
   render() {
     if (sessionStorage.jwt) {
       return (
         <div>
           <h1>Notifications</h1>
           <div className="feed">
-            {this.props.currentUser.notifications.map((n) => {
-              return <Notification notification={n} />;
+            {this.props.notifications.map((n) => {
+              return <Notification notification={n} key={n.id}/>;
```

Comment:
i'd say try to stay away from one letter variables: 
https://softwareengineering.stackexchange.com/questions/285288/is-the-use-of-one-letter-variables-encouraged

Reply:
obviously up for interpretation, but in my experience, the better the variable name, the easier time you'll have 6 months from now when you (or someone else) is trying to read what your thought process was